common/Cargo.toml

```diff
@@ -1,21 +1,21 @@
 [package]
 name = "xmtp_common"
-edition = "2021"
+edition = "2024"
 version.workspace = true
 license.workspace = true
 
 [dependencies]
 arc-swap.workspace = true
 async-trait.workspace = true
 bytes.workspace = true
-futures.workspace = true
+futures = { workspace = true, features = ["std"] }
 hex.workspace = true
 rand.workspace = true
+rand_chacha.workspace = true
 thiserror.workspace = true
-tokio = { workspace = true, features = ["time"] }
+tokio = { workspace = true, features = ["time", "rt"] }
 tracing.workspace = true
 web-time.workspace = true
-xmtp_cryptography.workspace = true
```

Comment:
removing cryptography from common means we don't pull in ethers which results in -300 dependencies or something. I did it for the compile times